sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -154,6 +155,14 @@ object ScalaNativePluginInternal {
       }
       libs
     },
+    nativeMode := "debug",
+    nativeLinkerReporter := tools.LinkerReporter.empty,
+    nativeOptimizerReporter := tools.OptimizerReporter.empty,
+    nativeLogger := streams.value.log,
+    nativeGC := "boehm"
+  )
```

Comment:
It seems like all settings/keys that we document should be here. Why are `nativeCompileOptions`, `nativeLinkingOptions`, `nativeClang`, `nativeClangPP` not there?